lib/utils/dataflow/src/index.js

```diff
@@ -622,14 +710,14 @@ const reducer = (reducefn, opt) => {
   // Configure our batch grouping reduction function
   const greduce = groupReduce(opt.input.type, yreducefn,
     opt.output.type, ocache, odb,
-    opt.sink.host, opt.sink.apps, opt.sink.post);
+    opt.sink.host, opt.sink.apps, opt.sink.post, edb);
```

Comment:
I'd suggest to move edb to just after odb, as errors are just an alternate type of output